api/envoy/api/v2/route/route.proto

```diff
@@ -175,6 +175,9 @@ message Route {
 
     // Return an arbitrary HTTP response directly, without proxying.
     DirectResponseAction direct_response = 7;
+
+    // Return an noop action 
```

Comment:
Please describe what it actually does, i.e its interaction with later routing, how request processing continues, etc.